suites/algorithms.js

```diff
@@ -1,20 +1,43 @@
+/*!
+ * Copyright (c) 2024 Digital Bazaar, Inc.
+ */
 import chai from 'chai';
 import {createInitialVc} from '../helpers.js';
 
 const expect = chai.expect;
 
-export function algorithmsAssertions({
-  endpoints,
-  testDescription,
-  vendorName,
+export function algorithmsSuite({
+  implemented,
+  testDescription = 'Data Integrity - Algorithms',
   credential,
+  features = {
+    authentication: false,
+    proofChain: false
+  }
 }) {
   return describe(testDescription, function() {
-    const columnId = testDescription;
+    // this will tell the report
+    // to make an interop matrix with this suite
+    this.matrix = true;
+    this.report = true;
+    this.rowLabel = 'Test Name';
+    this.columnLabel = 'Verifier';
```

Comment:
If we are hardcoding 'Verifier' here, shouldn't this be called `describeVerifier`?

Reply:
good catch here as the tests use both issuer and verifier this should be changed to `implementation` probably.

Reply:
this should have been fixed by an line comment.